src/models/users.js

```diff
@@ -92,12 +92,12 @@ const userSchema = new mongoose.Schema({
         type: Date,
         default: Date.now()
     },
-    address:[addressSchema],
-    avatar: string,
-    logbooks: [{ type: Schema.Types.ObjectId, ref: 'logblook' }]
+    address:{addressSchema},
```

Comment:
@AldaMayorgaDev Hay que quitar  las llaves vale

```suggestion
    address:addressSchema,
```

Ya que al pasarle el schema ya sería un objeto vale